client/src/components/AboutBanner/AboutBanner.css

```diff
@@ -0,0 +1,8 @@
+ul {
+    word-wrap: break-word;
+}
+
+img {
+    float: left;
```

Comment:
You can probably get away with:
`left: 0;`
instead of using floats. I'm not a fan of them and I find positional statements work better.